### ISSUE TITLE
Stats (and probably other areas): Word-breaking in some languages with larger font size

### ISSUE THREAD

**Issue by eduardozulian** [wordpress-mobile/WordPress-iOS#17994]
When you increase your font size and check Stats on some languages, longer words will break into two lines.

### Expected behavior

To read the whole word in a single line.

### Actual behavior

When translated, longer words are breaking due to lack of space.

![IMG_F274DCB233D4-1](https://user-images.githubusercontent.com/1136160/154807685-8cb9966c-e703-41a7-ad5e-d42409b90973.jpeg)

### Steps to reproduce the behavior
1. Change your iOS font size to something larger
2. Use a different language. For example, Italian or Portuguese
3. Open the WordPress app (if it was already opened, close it and open it again)
4. Go to My Site > Stats
5. Check the Views | Visitors | Likes | Comments section

##### Tested on iPhone 7, , iOS 15.1, WPiOS 19.2.0.1

---

**Comment by guarani** [comment]
This is reproducible on the new Traffic tab. We could consider:
- Not adapting to larger font sizes (the Health app does this for similar labels), and
- reduce font size to avoid word-breaking (if possible)

**Comment by staskus** [comment]
@irfano @guarani moving out a bug hunt. We should pay attention to this once we update/rebuild a Traffic tab.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
